### PR TITLE
feat(tutorial): complete step 4 - from sh bryan

### PR DESCRIPTION
**Pull request by supermqB**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/app.scss [hunk 1]
```diff
@@ -4,5 +4,8 @@
 @import 'carbon-components/scss/globals/scss/vars.scss';
 
 @import './components/TutorialHeader/tutorial-header.scss';
+@import './components/info/info.scss';
+
```
Comment (kellychurchill) [inline src/app.scss]: Should be `@import './components/Info/info.scss';`

File: src/app.scss [hunk 2]
```diff
@@ -4,5 +4,8 @@
...
 @import './content/LandingPage/landing-page.scss';
 @import './content/RepoPage/repo-page.scss';
+@import './components/info/info.scss';
```
Comment (kellychurchill) [inline src/app.scss]: Duplicate of line 7

File: src/app.scss (hunk 1 above)
Comment (supermqB) [reply src/app.scss]: changed it.

File: src/app.scss (hunk 2 above)
Comment (supermqB) [reply src/app.scss]: removed the duplicated line

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.

**Pull request closed without merging**